### DISCUSSION TITLE
Livewire filepond component not work on liveserver with https

### DISCUSSION THREAD

**Discussion by vanthao03596** [livewire/livewire#3407] [7. Bugs (archived)]
### Description
I'm using filepond component in livewire screencast. 
In local development everything work fine. But when go liveserver it not work. Progress in filepond not increment but livewire-tmp folder have file, it take a second to green. After call $this->photo->store('logo', 'public'), no error but the image not apeear in public disk.

**Context**
- Livewire version: 2.4.4
- Laravel version: 8.46.0
- Alpine version: 2.8.2
- Browser: Chrome

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)

**Comment by mokhosh** [comment]
REOPEN

**Comment by dario-valles** [comment]
You could solving by using like this:

`process: (fieldName, file, metadata, load, error, progress, abort, transfer, options) => {
                    @this.upload('upload', file, load, error,
                    (e) => {
                        progress(e.lengthComputable, e.loaded, e.total);
                    })
                }`

**Comment by mokhosh** [comment]
sweet.
i already had this based on Caleb's tutorial:
```js
process(fieldName, file, metadata, load, error, progress, abort, transfer, options) {
  @this.upload('{{ $attributes['wire:model'] }}', file, load, error, progress)
},
```
but it only worked on local.

thanks.
